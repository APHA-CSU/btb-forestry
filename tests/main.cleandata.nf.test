nextflow_process {

    name "Test Process cleandata"
    script "main.nf"
    process "cleandata"

    test("Should clean up input files") {

        when {
            params {
                today = "01Jan25"
            }
            process {
                """
                input[0] = file("${projectDir}/tests/data/subtest.csv")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
