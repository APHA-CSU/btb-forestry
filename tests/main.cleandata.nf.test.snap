{
    "Should clean up input files": {
        "content": [
            {
                "0": [
                    "bTB_Allclean_22Aug25.csv:md5,f22ea625845a3403b4f191fdfd90bd67"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-08-22T13:57:39.692948786"
    }
}